data-esnext.js

```diff
@@ -481,25 +481,32 @@ exports.tests = [
   }
 },
 {
-  name: 'weak references',
+  name: 'WeakReferences',
   spec: 'https://github.com/tc39/proposal-weakrefs',
-  category: STAGE2,
+  category: STAGE3,
   significance: 'large',
-  exec: function(){/*
-    var O = {};
-    var weakref = System.makeWeakRef(O);
-    var works = weakref.get() === O;
-    weakref.clear();
-    return works && weakref.get() === undefined;
-  */},
-  res : {
-    ie11: false,
-    firefox2: false,
-    opera10_50: false,
-    chrome65: false,
-    duktape2_0: false,
-    graalvm: false,
-  }
+  subtests: [
+    {
+      name: 'Weak references',
+      spec: 'https://github.com/tc39/proposal-weakrefs#weak-references',
+      exec: function(){/*
+        var O = {};
+        var weakref = new WeakRef(O);
+        var works = weakref.deref() === O;
+        weakref.clear();
```

Comment:
There is no such method as `WeakRef.prototype.clear`

Reply:
This is in the current test, I missed it in my revision.  How about splitting this test in two, one testing only the WeakRef() creation and other testing the persistence after a gc()?

Reply:
I couldn't make the gc() part work in Chrome, so I left it out.  I'll let someone more familiar with it write additional tests.